Examples/BasicExample/Makefile

```diff
@@ -1,21 +1,33 @@
+# Declares the root of the repository.
```

Comment:
I think it should be clear which repository you're talking about here, because this file is in the example, (and could have been copied out).
```suggestion
# Declares the root of the PlaydateKit repository.
```

Reply:
I think this might make developers more confused as to the difference between `PDKIT_ROOT` and `REPO_ROOT`. Maybe this can be changed to "this source code repository"?

Reply:
Wait, sorry, I think I got confused. If this is meant to be the root of the current project, what is it used for? I think you need to update the `include` line below to refer to `PDKIT_ROOT` also!

Reply:
Good catch, I'll update the include